mbed_greentea/mbed_report_api.py

```diff
@@ -17,6 +17,8 @@
 Author: Przemyslaw Wirkus <Przemyslaw.wirkus@arm.com>
 """
 
+from mbed_test_api import TEST_RESULT_OK
```

Comment:
One last nit pick! This line isn't necessary any more since you removed that last check. Mind pulling this out too?